python-connectors/salesforce-report/connector.py

```diff
@@ -29,30 +29,38 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
                       partition_id=None, records_limit=-1):
 
         results = self.client.make_api_call("/services/data/v39.0/analytics/reports/%s" % self.REPORT, parameters={"includeDetails": True})
-
-        report_format = results.get("reportMetadata").get("reportFormat")
-
-        if report_format != "TABULAR":
-            raise Exception("The format of the report is %s but the plugin only supports TABULAR." % report_format)
-
-        columns = results.get("reportMetadata").get("detailColumns", [])
-        log(columns)
-
-        log("records_limit: %i" % records_limit)
-
+        results_to_process = True
```

Comment:
It seems that `results_to_process` is equivalent to `if results`, it could be replaced by it